zerver/fixtures/bugdown-data.json

```diff
@@ -121,6 +121,12 @@
       "bugdown_matches_marked": true
     },
     {
+      "name": "escape",
+      "input": "\\**foo**",
+      "expected_output": "<p>**foo**</p>",
+      "bugdown_matches_marked": true
+    },
```

Comment:
the first "\" uses to escape the second one
and second one escapes first asterisk so it used as is, without interpreting as strong syntax